BibBuddy/app/src/main/java/de/bibbuddy/BookDao.java

```diff
@@ -340,5 +340,51 @@ public int countAllNotesForBook(Long bookId) {
     return noteCount;
   }
 
+  private Book createBookData(Cursor cursor) { // TODO use at other places after 2nd release
+
+    return new Book(
+        Long.parseLong(cursor.getString(0)),
+        cursor.getString(1),
+        cursor.getString(2),
+        cursor.getString(3),
+        Integer.parseInt(cursor.getString(4)),
+        cursor.getString(5),
+        cursor.getString(6),
+        cursor.getString(7),
+        cursor.getString(8),
+        Integer.parseInt(cursor.getString(9)),
+        Integer.parseInt(cursor.getString(10))
+    );
+  }
+
+
+  /**
+   * Finds all books which contain searchInput.
```

Comment:
```suggestion
   * Find all books that match the search input.
```

Reply:
I always wrote comments with s at the end of the first word.